cmd/microcloud/preseed.go

```diff
@@ -232,11 +305,20 @@ func (p *Preseed) validate(name string, bootstrap bool) error {
 		return fmt.Errorf("At least 2 systems are required to set up MicroCloud")
 	}
 
+	if !bootstrap && len(p.Systems) < 2 {
+		return fmt.Errorf("Missing a second system to add to MicroCloud")
+	}
+
+	systemNames := make([]string, 0, len(p.Systems))
 	for _, system := range p.Systems {
 		if system.Name == "" {
 			return fmt.Errorf("Missing system name")
 		}
 
+		if system.ListenAddress == "" {
+			return fmt.Errorf("Missing listen address for system %q", system.Name)
+		}
```

Comment:
This means ListenAddress must be specified for all systems, which defeats the purpose of mDNS, right?

Reply:
That is right.
As you anyway have to specify the `address` (formerly `listen_address`) for each system, there is no point in still using mDNS as we can reach out to the nodes directly based on their addresses.
The same happens in the interactive setup where MicroCloud asks you to set the address.

This is the replacement for the two top level keys `lookup_iface` and `lookup_subnet`.

Reply:
If the choice is between having 1 top-level `lookup_subnet` key that enables multicast over a subnet without directly specifying addresses, and outright disabling multicast for preseed, is there a reason you decided to go with the latter rather than the former?

Reply:
Okay, as discussed the `lookup_subnet` was added back in together with the `addrInSubnet` func that picks the first configured addressed out of the given subnet.